package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "bundle": "gulp bundle",
     "dev": "gulp preview",
+    "devlive": "export LIVERELOAD=true; gulp preview",
```

Comment:
Using `export` in the script is not portable: it generates errors on Windows when the npm script is run with PowerShell or CMD.
You could use the https://www.npmjs.com/package/cross-env or equivalent to make it portable.

Reply:
Issue fixed using the `cross-env` library.